lib/browser.ts

```diff
@@ -184,7 +185,7 @@ export class Protractor {
    *
    * @type {Plugins} Object containing plugin funtions from config.
    */
-  private plugins_: Plugins;
+  plugins_: Plugins;
```

Comment:
Hm yeah, these were fake private. We don't really want them as part of our documented API really, but the runner needs to access them. I'm fine making them public for now.